supersim_test.go

```diff
@@ -149,8 +149,11 @@ func createForkedInteropTestSuite(t *testing.T) *InteropTestSuite {
 	}
 }
 
-func createInteropTestSuite(t *testing.T) *InteropTestSuite {
-	testSuite := createTestSuite(t, &config.CLIConfig{})
+func createInteropTestSuite(t *testing.T, cliConfig *config.CLIConfig) *InteropTestSuite {
```

Comment:
what if instead of nil as a sentinel, we don't use a pointer argument for config? Passing nil around doesn't feel great.

Then it's just createInteropTestSuite(t, config.CLIConfig{})

```suggestion
func createInteropTestSuite(t *testing.T, cliConfig config.CLIConfig) *InteropTestSuite {
```